### DISCUSSION TITLE
Right Click Context Menus

### DISCUSSION THREAD

**Discussion by Tierran** [vkbo/novelWriter#1523] [Help]
I've been having an issue recently with my right click context menus. On the project content pane they simply don't show up at all. In the editor window, they can sometimes show up when I'm trying to do spelling corrections but it often takes many clicks and usually I can't get them to show. This is fairly recent behavior so I'm not sure why this is happening. I'm on 2.0.7 but it happens even on the 2.1.0 RC as well. Here is my system information:

![image](https://github.com/vkbo/novelWriter/assets/1091720/6e63c7bf-f2c7-41ac-957a-f48c38afda49)

Let me know if there are other dependencies I should be looking at.

**Comment by vkbo** [comment]
I've seen some erratic behaviour for context menus as well lately, but not consistently. I suspect it is due to changes on the window manager, because Qt keeps reporting errors when activating menus. Are you perhaps using Wayland now, and weren't before?

What I've seen is that they show up in a different location than requested, and it is possible they are off-screen for you. Both context menus are custom menus, but they use standard Qt functionality to open.

Do you have the same issue if you use `Ctrl+.` to open the menu? The shortcut should work both in the project tree and the editor.
